server/sonar-server/src/test/java/org/sonar/server/organization/ws/DeleteActionTest.java

```diff
@@ -362,7 +363,7 @@ public void delete_quality_gates() {
     verifyOrganizationDoesNotExist(org);
     assertThat(db.select("select uuid as \"uuid\" from quality_gates"))
       .extracting(row -> (String) row.get("uuid"))
-      .containsOnly(qualityGateInOtherOrg.getUuid());
+      .containsOnly(qualityGateInOtherOrg.getUuid(), qualityGateFinder.getBuiltInQualityGate(db.getSession()).getUuid());
```

Comment:
I don't understand => As no built-on has been created, this should fail, how does it work ???